src/icmp/IcmpSquid.cc

```diff
@@ -117,7 +120,7 @@ icmpSquidRecv(int, void *)
 void
 IcmpSquid::Recv()
 {
-    int n;
+    auto n = 0;
```

Comment:
Since you are touching `n` initialization please move its definition down to the place it is first set.
The `comm_udp_recv` line(s) should become:
```c++
    const auto n = comm_udp_recv(icmp_sock, static_cast<char *>(&preply), sizeof(pingerReplyData), 0);
```